app/repositories/application/OnlineTestRepository.scala

```diff
@@ -46,6 +46,8 @@ trait OnlineTestRepository {
 
   def insertOrUpdatePhase1TestGroup(applicationId: String, phase1TestProfile: Phase1TestProfile): Future[Unit]
 
+  def nextApplicationPendingExpiry: Future[Option[ExpiringOnlineTest]]
```

Comment:
No particular reason apart from the fact that that was the name in FastTrack.
Happy to change it though.